Building scheme "FBSnapshotTestCase tvOS" keeps failing

Travis log: https://travis-ci.org/danielgindi/Charts/builds/122282395

So it has been some weeks since iOS 9.3 was released. We notice Building scheme "FBSnapshotTestCase tvOS" keeps failing as below. We think it's FB's bug? Or any advice how to fix it?

```
37.38s$ cd Charts && carthage bootstrap

*** Cloning ios-snapshot-test-case

*** Skipped downloading ios-snapshot-test-case.framework binary due to the error:

	"GitHub API request failed: Maximum number of login attempts exceeded. Please try again later."

*** Checking out ios-snapshot-test-case at "2.1.0"

*** xcodebuild output can be found in /var/folders/my/m6ynh3bn6tq06h7xr3js0z7r0000gn/T/carthage-xcodebuild.QfaPgp.log

*** Building scheme "FBSnapshotTestCase tvOS" in FBSnapshotTestCase.xcworkspace

** BUILD FAILED **

The following build commands failed:

	Ld /Users/travis/Library/Developer/Xcode/DerivedData/FBSnapshotTestCase-bxqdwtzedguggmawxzyipupldvjd/Build/Products/Release-appletvos/FBSnapshotTestCase.framework/FBSnapshotTestCase normal arm64

(1 failure)

clang: error: linker command failed with exit code 1 (use -v to see invocation)

A shell task failed with exit code 65:

** BUILD FAILED **

The following build commands failed:

	Ld /Users/travis/Library/Developer/Xcode/DerivedData/FBSnapshotTestCase-bxqdwtzedguggmawxzyipupldvjd/Build/Products/Release-appletvos/FBSnapshotTestCase.framework/FBSnapshotTestCase normal arm64

(1 failure)
```

--- a/FBSnapshotTestCase.xcodeproj/project.pbxproj
+++ b/FBSnapshotTestCase.xcodeproj/project.pbxproj
@@ -482,6 +482,7 @@
 				DYLIB_CURRENT_VERSION = 1;
 				DYLIB_INSTALL_NAME_BASE = "@rpath";
 				EMBEDDED_CONTENT_CONTAINS_SWIFT = YES;
+				ENABLE_BITCODE = NO;
 				FRAMEWORK_SEARCH_PATHS = (
 					"$(PLATFORM_DIR)/Developer/Library/Frameworks",
 					"$(inherited)",
@@ -513,6 +514,7 @@
 				DYLIB_CURRENT_VERSION = 1;
 				DYLIB_INSTALL_NAME_BASE = "@rpath";
 				EMBEDDED_CONTENT_CONTAINS_SWIFT = YES;
+				ENABLE_BITCODE = NO;
 				FRAMEWORK_SEARCH_PATHS = (
 					"$(PLATFORM_DIR)/Developer/Library/Frameworks",
 					"$(inherited)",
@@ -746,6 +748,7 @@
 				8271378C1C63AB7000354E42 /* Release */,
 			);
 			defaultConfigurationIsVisible = 0;
+			defaultConfigurationName = Release;
 		};
 		827137901C63AB7000354E42 /* Build configuration list for PBXNativeTarget "FBSnapshotTestCase tvOS Tests" */ = {
 			isa = XCConfigurationList;
@@ -754,6 +757,7 @@
 				8271378E1C63AB7000354E42 /* Release */,
 			);
 			defaultConfigurationIsVisible = 0;
+			defaultConfigurationName = Release;
 		};
 		B31987EA1AB782D000B0A900 /* Build configuration list for PBXProject "FBSnapshotTestCase" */ = {
 			isa = XCConfigurationList;

